nixos/modules/installer/netboot/netboot.nix

```diff
@@ -28,7 +28,7 @@ with lib;
       ++ (if pkgs.stdenv.system == "aarch64-linux"
           then []
           else [ pkgs.grub2 pkgs.syslinux ]);
-    system.boot.loader.kernelFile = pkgs.stdenv.platform.kernelTarget;
+    system.boot.loader.kernelFile = pkgs.stdenv.hostPlatform.platform.kernelTarget;
```

Comment:
this might be removed because it is default anyway

Reply:
I'll do that separately, one change at a time :). But thanks for catching!